pkgs/build-support/replace-secret/replace-secret.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, python3 }:
+
+stdenv.mkDerivation {
+  name = "replace-secret";
+  buildInputs = [ (python3.withPackages (p: [ p.click ]))];
```

Comment:
Is it possible to make this package not depend on `click` perhaps?

Reply:
Sure, I switched to `argparse` instead.